wagtailcache/wagtail_hooks.py

```diff
@@ -43,3 +44,15 @@ def register_cache_menu():
         reverse("wagtailcache_admin:index"),
         classnames="icon icon-" + CACHE_ICON,
     )
+
+
+@hooks.register("after_create_page")
+@hooks.register("after_edit_page")
+def clear_wagtailcache(request, page):
```

Comment:
This is good... although I think it should be added as a code snippet in the docs, rather than in the package. That way the user can choose if they want it or not. You can add it aside the other example in: https://github.com/coderedcorp/wagtail-cache/blob/main/docs/getting_started/usage.rst

Reply:
@vsalvino 
we do it this way. I just forgot to take it out.